Update model.rst (#3039)
hyperspy/hyperspy: doc/user_guide/model.rst

reduce the starting value (or local / false minima) problem, which often arises
when fitting multi-dimensional datasets.

> The algorithm is described in full in "Multi-dimensional Data Analysis in Electron Microscopy" by Tomas Ostasevicius.
> This thesis is available in the following `thesis
> <https://doi.org/10.17863/CAM.15486>`_.

Reviewer: ```suggestion
The algorithm is described in `Tomas Ostasevicius' PhD thesis <https://doi.org/10.17863/CAM.15486>`_, entitled  "Multi-dimensional Data Analysis in Electron Microscopy".
```
